frontend/public/components/dashboard/capacity-card/capacity-item.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { ChartDonut, ChartThemeColor, ChartThemeVariant, ChartTooltip } from '@patternfly/react-charts';
+
+import { USED, AVAILABLE, LOADING, CAPACITY, UNKNOWN, NOT_AVAILABLE } from './strings';
+import { LoadingInline, Humanize } from '../../utils';
+
+export const CapacityItem: React.FC<CapacityItemProps> = React.memo(({ title, used, total, formatValue, isLoading = false }) => {
+  let description;
+  let data = [{ x: USED, y: 0, label: null }, { x: AVAILABLE, y: 100, label: null }];
+  let primaryTitle;
+  let secondaryTitle = CAPACITY;
+  if (isLoading) {
+    description = <LoadingInline />;
+    primaryTitle = LOADING;
+  } else if (used == null || total == null) {
+    description = NOT_AVAILABLE;
+    primaryTitle = UNKNOWN;
+  } else {
+    const totalFormatted = formatValue(total);
+    const usedFormatted = formatValue(used, null, totalFormatted.unit);
+
+    const available = formatValue(totalFormatted.value - usedFormatted.value, totalFormatted.unit, totalFormatted.unit);
+    const percentageUsed = total > 0 ? Math.round((100 * usedFormatted.value) / totalFormatted.value) : 0;
+
+    data = [
+      {
+        x: USED,
+        y: usedFormatted.value,
+        label: `${USED}: ${usedFormatted.string}`,
+      },
+      {
+        x: AVAILABLE,
+        y: available.value,
+        label: `${AVAILABLE}: ${available.string}`,
+      },
+    ];
+    description = (
+      <>
+        <span className="co-capacity-card__item-description-value">{available.string}</span>
+        {' available out of '}
+        <span className="co-capacity-card__item-description-value">{totalFormatted.string}</span>
+      </>
+    );
+    primaryTitle = `${percentageUsed.toString()}%`;
+    secondaryTitle = USED;
+  }
+  return (
+    <div className="co-capacity-card__item">
+      <div className="co-capacity-card__item-title">{title}</div>
+      <h6 className="co-capacity-card__item-description">{description}</h6>
+      <div id={_.uniqueId('donut-chart-')} className="co-capacity-card__item-graph">
+        <ChartDonut
```

Comment:
PF4 Charts has a new component called `ChartDonutUtilization` which would probably be appropriate to use here. http://patternfly-react.surge.sh/patternfly-4/charts/chartdonututilization/

I'm also currently working on migrating our `Gauge` component to use `ChartDonutUtilization`. Once completed, , that component would also probably be appropriate here.

Reply:
We will need to update the version of PF4 charts dep because the current one does not have `ChartDonutUtilization`.

I will be happy to reuse the one you are working on once it lands. Could we do that as a follow-up ?

Reply:
@TheRealJon ping

Reply:
@rawagner Yeah, that sounds good. In the meantime, do you mind adding a colorScale prop `colorScale={['#06C', '#EDEDED'])` so that the colors for used/unused portions look a bit more like the ChartDonutUtilization?

Reply:
Not at all :) I've added `colorScale` in latest update

![c_scale](https://user-images.githubusercontent.com/2078045/59780840-f005e000-92ba-11e9-9d53-77f654c723a8.png)